backend/api/tests/views/conftest.py

```diff
@@ -12,22 +10,15 @@
 from api.tests.common import AuthenticatedClient
 
 _CHANNEL_NAME: Final[str] = "mychannel"
-_EXTRA_HTTP_HEADERS: Final[dict[str, str]] = {"HTTP_SUBSTRA_CHANNEL_NAME": _CHANNEL_NAME}
```

Comment:
What is the rationale for getting rid of this header? Does the client fill it on on its own when `client.channel` is set?

Reply:
since the AuthenticatedClient now has a channel by default ('mychannel') this header (as well as every other headers) are now redundant, as long as you use the AuthenticatedClient, other kinds of Client in the tests still need similar headers